src/Wlan.cpp

```diff
@@ -160,7 +160,12 @@ void handleWifiStateInit() {
 	connectionAttemptCounter = 0;
 	connectStartTimestamp = 0;
 	connectionFailedTimestamp = 0;
-	wifiState = WIFI_STATE_CONNECT_LAST;
+	#ifdef ALWAYS_SCAN_WIFI_ON_STARTUP
```

Comment:
As discussed please use a NVS setting here, e.g. 
`bool performScan = gPrefsSettings.getBool("ALWAYS_SCAN_WIFI_ON_STARTUP", true);`

Reply:
Will do so.
Where can we modify this option afterwards? Shall we add a checkbox in the wifi-settings in the Webserver for that or would you just leave it that way for now?

Reply:
I would preset performScan with true, so at default ESPuino startup performs a scan and always connects to stronger WiFi (same name).

We could add this setting later in Web-UI. At this moment we can live without it ;-)
Just my opinion..